controller/src/main.cpp

```diff
@@ -97,3 +95,7 @@ void setup() {
 
   controller_loop();
 }
+
+void loop() {
+  // Dummy placeholder as Arduino framework requires it to compile
```

Comment:
Should we just move controller_loop() into the loop() function?

Reply:
Maybe. But then we don't control how it's called. Main function looks like this in ArduinoFramework right now:

`int main(void)
{
	init();

	initVariant();

#if defined(USBCON)
	USBDevice.attach();
#endif
	
	setup();
    
	for (;;) {
		loop();
		if (serialEventRun) serialEventRun();
	}
        
	return 0;
}`